project2/index.html

```diff
@@ -0,0 +1,98 @@
+<html lang="en">
```

Comment:
You're missing the `doctype` declaration at the top of your document
```suggestion
<!DOCTYPE html>
<html lang="en">
```